source/config/__init__.py

```diff
@@ -76,6 +82,7 @@ def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	outputDevice = string(default=default)
 	autoLanguageSwitching = boolean(default=true)
 	autoDialectSwitching = boolean(default=false)
+	readNumbersAs=integer(default=0,min=0,max=3)

```

Comment:
nit: Please add space either side of the first "=" sign for consistency with the rest of the options.